src/data/faq.json

```
@@ -116,6 +116,62 @@
                     }
                 ]
             },
+            {
+                "title": "Testzertifikat",
+                "id": "test_cert",
+                "accordion": [
+                    {
+                        "title": "What is the purpose of the digital COVID test certificate?",
+                        "anchor": "test_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "Starting with version 2.4 of the Corona-Warn-App (CWA), you can request a digital test certificate via the app.  The new feature allows you to prove your negative test result through an official, digital COVID test certificate in the form of a QR code in the CWA.  Within the countries of the European Union, it can serve as officially valid proof of a negative test result, which you can use, for example, for travel or in other legally prescribed cases, such as a visit to a restaurant. You can then show the QR code in the CWA."
+                        ]
+                    },
+                    {
+                        "title": "How can I request a test certificate?",
+                        "anchor": "test_cert_get",
+                        "active": true,
+                        "textblock": [
+                            "You can request the certificate by registering a test in the Corona-Warn-App (CWA): To do this, you can tap on 'Register Test' on the star screen of your CWA and then on 'Scan QR Code'. After scanning the QR code for test registration, the window for the COVID test certificate automatically opens. There you  can 'request the test certificate' or reject it if you do not want to receive one."

```

"star screen" => "start screen"

---

Again an empty space too much and one suggestions:

```suggestion
                            "You can request the certificate by registering a test in the Corona-Warn-App (CWA): To do this, tap on 'Register Test' on the star screen of your CWA and then on 'Scan QR Code'. After scanning the QR code for test registration, the window for the COVID test certificate automatically opens. There you can 'request the test certificate' or reject it if you do not want to receive one."
```

Please note that the typo found by @MikeMcC399 is not included in this suggestion and has to be corrected independently.